### ISSUE TITLE
Place: adding an accessibility note triggers an error message appearing in the Note

### ISSUE THREAD

**Issue by kmdvs** [culturecreates/footlight-app#1789]
When creating a new Place, or modifying an existing Place: 
- adding an accessibility Note displays the Note with an error message inside the text field (greyed text); and you cannot add a note

![Image](https://github.com/user-attachments/assets/829e694c-b539-4ca4-a828-224427e67d59)

Expected behaviour: Note field functions correctly.

Addition from Caitlin: Please investigate thoroughly and make sure that errors do not appear in any other entities and their Note fields.

**Comment by sahalali** [comment]
@AbhishekPAnil, could you please prioritize this issue as it is currently in production?

**Comment by AbhishekPAnil** [comment]
There was a translation object issue in the translation file :
      "placeholder": { 
                 "english": "Ajouter une note en anglais", 
                 "french": "Ajouter une note en français" 
               },

      I have changed it to :

**Comment by AbhishekPAnil** [comment]
>       There was a translation object issue in the translation file :
>       "placeholder": { 
>                  "english": "Ajouter une note en anglais", 
>                  "french": "Ajouter une note en français" 
>                },
> 
>       I have changed it to :
>               "placeholder": "Entrer une note",

The english translation for the same was to"Enter a note"
Please let me know if you want different translation.
